site/plugins/site/composer.json

```diff
@@ -1,11 +1,15 @@
 {
-    "name": "kirby/site",
-    "autoload": {
-        "psr-4": {
-            "Kirby\\Site\\": "src/"
-        }
-    },
-    "require": {
-        "phpdocumentor/reflection-docblock": "^3.1"
+  "name": "getkirby/site",
+  "autoload": {
+    "psr-4": {
+      "Kirby\\": "src/"
     }
+  },
+  "require": {
+    "algolia/algoliasearch-client-php": "^1.25",
```

Comment:
Can we switch to v2? Because v1 doesn't support PHP8.

Reply:
I only migrated the existing plugin into a new structure. I would see upgrading (and making changes so the plugin still works) as a separate little project after merging this.